packages/api/src/record.ts

```diff
@@ -162,4 +166,11 @@ export class IPNSRecord {
     }
     return Date.now() + interval
   }
+
+  get publisherEndpointToken (): string {
+    if (this.env.PUBLISHER_AUTH_SECRET && this.env.PUBLISHER_AUTH_SECRET !== '') {
```

Comment:
Isn't the second half of this `if` condition unnecessary? If the value is empty then the first condition will be false-y, right?

Reply:
ah true with strings

Reply:
```suggestion
    if (this.env.PUBLISHER_AUTH_SECRET) {
```